src/screens/AdminPortal/SubTags/SubTags.spec.tsx

```
@@ -1,7 +1,14 @@
 import React from 'react';
 import { MockedProvider } from '@apollo/react-testing';
 import type { RenderResult } from '@testing-library/react';
-import { act, cleanup, render, screen, waitFor } from '@testing-library/react';
+import {
+  act,
+  cleanup,
+  fireEvent,
```

ESLint

[error] 'fireEvent' import from '@testing-library/react' is restricted. Tests in this file use fireEvent for user interactions; use userEvent from `@testing-library/user-event` instead.

(no-restricted-imports)